tools/check-wininstaller-translations.sh

```diff
@@ -11,7 +11,7 @@ for LANGUAGE_FILE in ${LANGUAGE_FILES}; do
     echo
     echo "* ${LANGUAGE_FILE}"
     echo -n "  - Checking language file is included in ${INSTALLERLNG}... "
-    if grep -q '^!include "\${ROOT_PATH}\\'$(tr '/' '\\' <<<"${LANGUAGE_FILE}" | sed -re 's|\\|\0\0|g')'"' "${BASE_DIR}/${INSTALLERLNG}"; then
+    if grep -q '^!include "\${ROOT_PATH}\\'$(sed -re 's|/|\\\\|g' <<<"${LANGUAGE_FILE}")'"' "${BASE_DIR}/${INSTALLERLNG}"; then
```

Comment:
I see how getting rid of the "backslash to nul character" transformation makes sense, but it looks like replacing `tr` with `sed` does the opposite of simplifying.

Reply:
> but it looks like replacing tr with sed does the opposite of simplifying.

I agree that it still doesn't look nice, but also consider that it replaces one `tr` + non-nice `sed` with just a single (still) non-nice `sed`. @softins initially proposed this and I think it does simplify, but if there's agreement that it does not, then it's fine for me as well.

This was initially proposed here: https://github.com/jamulussoftware/jamulus/pull/1203/files#diff-0307ece3ec605715410994dc14469df63db22548ac539792e241b7f561d6a918R14


This is really just a helper script for the maintainers and nothing in the Jamulus core. Maybe we should even move these into their own repo?

Reply:
I don't believe that is a null character. It looks to me like \0 in that context means "what was matched", the same as & would. I can't see how it would work otherwise.

I meant to reply earlier. My suggestion was only from the point of view that it seemed more efficient to do `/` > sed > `\\` than doing `/` > tr > `\` > sed > `\\`

But it's only a micro-optimisation.

Reply:
> I don't believe that is a null character. It looks to me like \0 in that context means "what was matched", the same as & would. I can't see how it would work otherwise.

Yes, that was the intention and that's how it works.

Reply:
> I don't believe that is a null character. It looks to me like \0 in that context means "what was matched", the same as & would.

Oh, you're right. Never mind then.